src/main/java/com/o19s/es/explore/StatisticsHelper.java

```diff
@@ -84,4 +84,20 @@ public float getStdDev() {
 
         return (float) Math.sqrt(getVariance());
     }
+
+    public float getAvg() {
```

Comment:
Does this logic differ from calling getMean()?  We may want to add some comments explaining the difference.

Reply:
I dind`t see `getMean` 😔. We will fix it